server/src/main/java/org/opensearch/rest/action/cat/RestThreadPoolAction.java

```diff
@@ -267,6 +271,7 @@ private Table buildTable(RestRequest req, ClusterStateResponse state, NodesInfoR
                 table.addCell(poolStats == null ? null : poolStats.getRejected());
                 table.addCell(poolStats == null ? null : poolStats.getLargest());
                 table.addCell(poolStats == null ? null : poolStats.getCompleted());
+                table.addCell(poolStats == null ? null : poolStats.getWaitTimeNanos());
```

Comment:
The cell should contain just a numeric value of nanos, otherwise it may be difficult to scape it (scripts or monitoring) since the time unit needs to be parsed

Reply:
This is only for the `_cat` API, which is meant only for human consumption anyways. The `search.query_time` metrics all use this `TimeValue` output:

https://github.com/opensearch-project/OpenSearch/blob/079c3e9c95eccd26ceb26f36a7c7d0a69b6af1e2/server/src/main/java/org/opensearch/rest/action/cat/RestNodesAction.java#L549

Reply:
> So I think it's better to keep this as is since it's meant to be human readable anyways.

Sure, let's keep it consistent for CAT APIs